.gitignore

```diff
@@ -27,3 +27,8 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+
+#Jenkins X specific files
+jx-requirements.yml
+kubeconfig_*
```

Comment:
newline missing ;-)

Reply:
btw, why is there an _ at the end of kubeconfig? Any reason we write out _kubeconfig_?

Reply:
apropos .gitignore, it seems a `.DS_Store` has made it into version control. Should be removed and `.DS_Store` added to .gitignore